runners/java-fn-execution/src/test/java/org/apache/beam/runners/fnexecution/control/RemoteExecutionTest.java

```diff
@@ -510,30 +514,33 @@ public void testMetrics() throws Exception {
                 ParDo.of(
                     new DoFn<byte[], String>() {
                       private boolean emitted = false;
-                      Counter startCounter =
+                      private Counter startCounter =
                           Metrics.counter(RemoteExecutionTest.class, startUserCounterName);
 
                       @StartBundle
                       public void startBundle() throws InterruptedException {
+                        Thread.sleep(1000);
```

Comment:
any way for us to use smaller values here, or avoid Thread.sleep at all?

Reply:
The one way we could remove these sleeps is by exposing the metrics during the test, and polling until we see them. However, the metrics are only available when the bundle finishes. 

I filed BEAM-6597 with a TODO on how to do this and upgrade this test.

I don't want to go much lower than this, for fear of making this test noisy/flakey.